x-pack/legacy/plugins/rollup/__jest__/client_integration/job_create_clone.test.js

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { setupEnvironment, pageHelpers } from './helpers';
+import { JOB_TO_CLONE, JOB_CLONE_INDEX_PATTERN_CHECK } from './helpers/constants';
+
+jest.mock('ui/index_patterns', () => {
+  const { INDEX_PATTERN_ILLEGAL_CHARACTERS_VISIBLE } = require.requireActual(
+    '../../../../../../src/legacy/ui/public/index_patterns/constants'
+  );
+  return { INDEX_PATTERN_ILLEGAL_CHARACTERS_VISIBLE };
+});
+
+jest.mock('ui/chrome', () => ({
+  addBasePath: () => '/api/rollup',
+  breadcrumbs: { set: () => {} },
+  getInjected: () => ({}),
+}));
+
+jest.mock('lodash/function/debounce', () => fn => fn);
+
+const { setup } = pageHelpers.jobClone;
+const {
+  jobs: [{ config: jobConfig }],
+} = JOB_TO_CLONE;
+
+describe('Cloning a rollup job through create job wizard', () => {
+  let httpRequestsMockHelpers;
+  let server;
+  let find;
+  let exists;
+  let form;
+  let table;
+  let actions;
+
+  beforeAll(() => {
+    ({ server, httpRequestsMockHelpers } = setupEnvironment());
+  });
+
+  beforeEach(() => {
+    httpRequestsMockHelpers.setIndexPatternValidityResponse(JOB_CLONE_INDEX_PATTERN_CHECK);
+
+    ({ exists, find, form, actions, table } = setup());
+  });
+
+  afterAll(() => {
+    server.restore();
+  });
+
+  it('should have fields correctly pre-populated', async () => {
```

Comment:
nit: it may be helpful to add comments indicating when you're on a different step, or break this up into smaller `it()` statements.

Reply:
Added comments :+1: